test/e2e/report_dynamic_input_data_test.go

```diff
@@ -151,7 +158,7 @@ func testReportsProduceData(t *testing.T, testReportingFramework *reportingframe
 
 			t.Logf("reportStart: %s, reportEnd: %s", reportStart, reportEnd)
 
-			report := test.newReportFunc(test.name, test.queryName, test.schedule, &reportStart, &reportEnd)
+			report := test.newReportFunc(test.name, test.queryName, test.schedule, &reportStart, &reportEnd, nil)
```

Comment:
Is the nil needed?

Reply:
Yes, because I updated `newReportFunc` to take an expiration so all the pre-existing usages needed to be updated and "Go does not have optional parameters nor does it support method overloading"